ngraph/core/include/ngraph/op/gather.hpp

```diff
@@ -61,5 +61,32 @@ namespace ngraph
                     clone_with_new_inputs(const OutputVector& new_args) const override;
             };
         } // namespace v7
+
+        namespace v8
```

Comment:
Do we need to update specification?

Reply:
yes, here is the spec for G8
https://github.com/openvinotoolkit/openvino/pull/6172